DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -1240,12 +1240,16 @@ internal async Task OnGuildMemberRemoveEventAsync(TransportUser user, DiscordGui
         internal async Task OnGuildMemberUpdateEventAsync(TransportMember member, DiscordGuild guild)
         {
             var userAfter = new DiscordUser(member.User) { Discord = this };
-            userAfter = this.UpdateUserCache(userAfter);
+            _ = this.UpdateUserCache(userAfter);
```

Comment:
@VelvetThePanda, could you verify that this won't turn into #1257?

Reply:
All UpdateUserCache does is poly-fill. I see no reason for this to break.